controllers/openshift_controller.go

```diff
@@ -936,6 +952,7 @@ func (r *KataConfigOpenShiftReconciler) processKataConfigInstallRequest() (ctrl.
 	}
 
 	// peer pod enablement
+	// This should be done after kata-oc creation and kata installation
```

Comment:
Wouldn't this be executed _before_ kata-oc creation on the first reconciliation after a KataConfig instance is created?

Reply:
This is an incorrect comment. I'll remove it.